gui/app/modes/CommandPressureMode.qml

```diff
@@ -24,285 +24,185 @@ Mode {
         }
     }
 
-    ColumnLayout {
-        id: columnLayout
-        anchors.fill: parent
-
-        RowLayout {
-            id: rowLayout
-            width: 100
-            height: 100
-
-            ColumnLayout {
-                id: columnLayout1
-                width: 100
-                height: 100
-                Layout.maximumHeight: 65356
-                Layout.maximumWidth: 6553
+    Rectangle {
+        id: parameterDisplayPanel
+        width: 216; height: 440
+        color: "#05121C"
+        radius: 8
+        anchors {
+            top: parent.top;
+            right: parent.right; rightMargin: 8
+        }
+
+        GridLayout {
+            anchors {
+                top: parent.top
+                horizontalCenter: parent.horizontalCenter
+            }
+            columns: 2
+            columnSpacing: 0
+            rowSpacing: 0
+
+            ParameterDisplay {
+                parameterName: qsTr("PiP")
```

Comment:
It's capitalized as PIP

Reply:
The design reference says "PiP", should I ignore it?
See:
https://xd.adobe.com/view/606e962c-020f-4b23-7140-53ea2fe6b21a-165f/screen/7f791a1d-21c0-43ba-8f6a-8ae214efb5be/xx-Modes